lib/App/DuckPAN.pm

```diff
@@ -275,6 +275,17 @@ sub _build_ddg {
 	App::DuckPAN::DDG->new( app => shift );
 }
 
+has fathead => (
```

Comment:
This allows DuckPAN to load the `App::DuckPAN::Fathead` into app.

Combined with `HasAPP` it enables access to the Fathead lib where needed, via `$self->app->fathead->`.

I was contemplating renaming the lib to something like `App::DuckPAN::Util::Fathead` or similar. We load Perl.pm and DDG.pm in the same fashion for the same reasons.